wm/desk.go

```diff
@@ -241,7 +240,8 @@ func (x *x11WM) runLoop() {
 			}
 		case xproto.LeaveNotifyEvent:
 			if mouseNotify, ok := desktop.Instance().(notify.MouseNotify); ok {
-				mouseNotify.MouseInNotify(int(ev.RootX), int(ev.RootY))
+				mouseNotify.MouseInNotify(fyne.NewPos(int(float32(ev.RootX)/fyne.CurrentApp().Settings().Scale()),
```

Comment:
This will not work with auto scale - the settings just stores the preference not the canvas scale